### ISSUE TITLE
Direct Deposit 508 Audit

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#1329]
This is the parent container for any 508 findings from the Product Team audit of Direct Deposit.

Please see the [severity rubric for 508 issues](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/Practice%20Areas/Accessibility/GUIDANCE-severity-rubric.md) on Github. This document will help organize and triage 508 issue tickets.

**Comment by 1Copenut** [comment]
@Justin-Pickett I've completed the audit of the direct deposit app. Let me know when issues are fixed and you're ready for me to send to the VA 508 office. 

I generally like to have all issues remediated before sending, but can move it up if launch is happening soon and let them know about our findings.

Thank you for the updates as this feature moved through the pipeline!

**Comment by Justin-Pickett** [comment]
@1Copenut thank you. Will do!

**Comment by lkoenigsberg** [comment]
@Justin-Pickett , it seems all issues related to this epic are done and maybe it should be closed?
